packages/gatsby/content/advanced/contributing.md

```diff
@@ -122,6 +122,11 @@ If it fails and you have no idea why, feel free to ping a maintainer and we'll d
 
 **Note:** If you modify one of the [default plugins](https://github.com/yarnpkg/berry#default-plugins), you will also need to bump `@yarnpkg/cli`.
 
+## Reviewing other PRs
+
+You're welcome to leave comments if you spot glaring bugs, but do not approve PRs if you're not a member.
+It's generally seen as [bad form](https://twitter.com/brian_d_vaughn/status/1224051534536667137) in the open source community.
+
```

Comment:
I think it would be better for this section to be the last one since it isn't something we encounter on a regular basis.

Reply:
This section currently put after preparing your PR section. Isn't that a better location as the document flow is talking about PRs?

Reply:
:thinking: In this case, perhaps it would make more sense to move `Writing documentation` right before `Preparing your PR to be released`

Reply:
I'm a new contributor and do not know why writing documentation was added in the end. Could it be because it's the not the documentation directly is written in code, but in different package gatsby?

I don't think it should block this PR though.

Reply:
(Apologies for the late reply)

> do not know why writing documentation was added in the end

Neither do I since that happened 3 years ago in https://github.com/yarnpkg/berry/pull/307 before my involvement in the project, so let's ping @arcanis for details.

> Could it be because it's the not the documentation directly is written in code, but in different package gatsby?

Perhaps, but since writing documentation is something people do more often than contributing code, I feel like it would make more sense to not have it be after a section that few people find important.

> I don't think it should block this PR though.

The ordering of the sections is still important as IMO a section as important as `Writing documentation` shouldn't be buried after `Reviewing other PRs`.

"Blocking" is a bit of a strong word in this situation as, while it would be nice to have this resolved before merging this PR, it can also be done in a follow-up PR (i.e. I won't block on this if you feel strongly about not reordering the sections yourself).

Another idea I've thought about: perhaps `Writing documentation` could instead be moved right before `Writing your feature` since there are generally more people contributing documentation than features. :thinking:

Reply:
I created an issue to revisit ordering of sections in CONTRIBUTING at https://github.com/yarnpkg/berry/issues/4098
The decision can be taken async by yarnpkg team.